components/auth/AuthProvider.tsx

```diff
@@ -62,36 +60,46 @@ export const AuthProvider = () => {
 const fetchData = async (user: Privy.User, wallets: Privy.ConnectedWallet[]) => {
   console.log("AuthProvider.fetchData");
 
-  const token = await Privy.getAccessToken();
-  if (!token) {
-    return ToastSender.Error("Haha, and you thought this would be easy!?");
-  }
-
   try {
-    await UserCreate(token, [{ image: "", name: truncateEthAddress(wallets[0]?.address) }]);
-  } catch (err) {
-    return ToastSender.Error("Haha, and you thought this would be easy!?");
-  }
+    const token = await Privy.getAccessToken();
+    if (!token) {
+      return ToastSender.Error("Your access token could not be validated.");
+    }
 
-  const auth = {
-    id: "",
-    image: "",
-    name: "",
-    token: token,
-    valid: true,
-    wallet: user?.wallet?.address || "",
-  };
+    // TODO setup smart account client
```

Comment:
Ignore this. This here comes next. The changes were stashed and I just went with it.